R/learner_xgboost_surv_xgboost_cox.R

```diff
@@ -142,8 +137,10 @@ LearnerSurvXgboostCox = R6Class("LearnerSurvXgboostCox",
   private = list(
     .train = function(task) {
       pv = self$param_set$get_values(tags = "train")
+      # manually add 'objective' and 'eval_metric'
+      pv = c(pv, objective = "survival:cox", eval_metric = "cox-nloglik")
```

Comment:
if pv is empty I think this will construct a character vector and not a list

Reply:
Personally, I would not remove the eval metric (I might want to optimizer objective1 but also keep track of objective2)